src/Core/Framework/Api/Command/DumpClassSchemaCommand.php

```diff
@@ -106,13 +116,19 @@ private function getCollectionEntity(string $className): ?string
                 /** @var ClassConstFetch $classConst */
                 $classConst = $returnStatement->expr;
 
+                /** @phpstan-ignore cast.string (it can be either PhpParser\Node\Expr or PhpParser\Node\Name) */
                 return (string) $classConst->class;
             }
         }
 
         throw new \InvalidArgumentException(\sprintf('Invalid class given %s', $className));
     }
 
+    /**
+     * @param Stmt[] $stmts
```

Comment:
`Stmt[]` can also be `array<string, Stmt>` or `array<int, Stmt>`. sometimes it is `list<Stmt>` 

would be nice if you could be more precise on all occurrences :slightly_smiling_face: